tests/dm_syncer/conf/dm-syncer-2.toml

```diff
@@ -0,0 +1,82 @@
+# Old Configuration.
+
+# used for binlog replication 
+server-id = 101
+
+# which flavor mysql/mariadb
+flavor = "mysql"
+
+# meta schema in downstreaming database to store meta informaton of dm
+meta = "/tmp/dm_test/dm_syncer/old_meta_file"
```

Comment:
How about setting this line in _run.sh_ dynamically?

Reply:
ok, good idea